README.md

```diff
@@ -69,10 +69,10 @@ https://docs.python.org/3/library/argparse.html#formatter-class):
 The default styles used by *rich-argparse* are carefully chosen to work in different light and dark
 themes.
 
-### Customize the colors
+### Customize the font style and colors
```

Comment:
This is not technically correct. The word "style" here is the rich definition of a style (see https://rich.readthedocs.io/en/latest/style.html) which includes colors, and various other attributes like bold, italic, etc. It also includes background colors that have nothing to do with a "font style".
I suggest keeping the old text as it is simple enough for a section title